package.json

```diff
@@ -213,7 +213,7 @@
     "node-unzip-2": "^0.2.8",
     "oclif": "^1.15.2",
     "open": "^7.0.2",
-    "patch-package": "^6.2.0",
+    "patch-package": "6.1.2",
```

Comment:
I have submitted an issue and PR upstream:
* https://github.com/ds300/patch-package/issues/201
* https://github.com/ds300/patch-package/pull/202